### ISSUE TITLE
Installation via Homebrew fails

### ISSUE THREAD

**Issue by Dschee** [yonaskolb/Beak#16]
Currently the installation of beak via Homebrew fails. This was reported to me, so I removed beak from Homebrew and got the same error (and can't use Beak anymore myself 😭 ):

```
==> Installing beak from yonaskolb/beak
==> Downloading https://github.com/yonaskolb/Beak/archive/0.3.2.tar.gz
Already downloaded: /Users/Cihat/Library/Caches/Homebrew/beak-0.3.2.tar.gz
==> Building Beak
==> swift build --disable-sandbox -c release -Xswiftc -static-stdlib
Last 15 lines from /Users/Cihat/Library/Logs/Homebrew/beak/01.swift:
error: product dependency 'Utility' not found
error: product dependency 'SwiftShell' not found
error: product dependency 'Spectre' not found
error: product dependency 'PathKit' not found
Fetching https://github.com/jpsim/SourceKitten
Fetching https://github.com/kylef/PathKit.git
Fetching https://github.com/kylef/Spectre.git
Fetching https://github.com/yonaskolb/swift-package-manager
Fetching https://github.com/kareman/SwiftShell.git
Fetching https://github.com/drmohundro/SWXMLHash.git
Fetching https://github.com/jpsim/Yams.git
Fetching https://github.com/norio-nomura/Clang_C.git
Fetching https://github.com/Carthage/Commandant.git
Fetching https://github.com/antitypical/Result.git
Fetching https://github.com/norio-nomura/SourceKit.git

If reporting this issue please do so to (not Homebrew/brew or Homebrew/core):
yonaskolb/beak

/usr/local/Homebrew/Library/Homebrew/utils/github.rb:220:in `raise_api_error': curl failed!  (GitHub::Error)
curl: (22) The requested URL returned error: 422 Unprocessable Entity
curl: (3) <url> malformed
	from /usr/local/Homebrew/Library/Homebrew/utils/github.rb:178:in `open'
	from /usr/local/Homebrew/Library/Homebrew/utils/github.rb:278:in `search'
	from /usr/local/Homebrew/Library/Homebrew/utils/github.rb:225:in `search_issues'
	from /usr/local/Homebrew/Library/Homebrew/utils/github.rb:238:in `issues_for_formula'
	from /usr/local/Homebrew/Library/Homebrew/exceptions.rb:369:in `fetch_issues'
	from /usr/local/Homebrew/Library/Homebrew/exceptions.rb:365:in `issues'
	from /usr/local/Homebrew/Library/Homebrew/exceptions.rb:419:in `dump'
	from /usr/local/Homebrew/Library/Homebrew/brew.rb:138:in `rescue in <main>'
	from /usr/local/Homebrew/Library/Homebrew/brew.rb:30:in `<main>'
```

**Comment by yonaskolb** [comment]
Thanks for raising this @Dschee! 
It was an issue in the dependency tree, as the Package was pointing at a commit that no longer existed due to a rebase. That and the fact that homebrew fails with a nonsensical error if a tapped formula fails.
Fixed in #17

**Comment by yonaskolb** [comment]
Fixed in 0.3.3 👍

**Comment by Dschee** [comment]
Thanks for fixing quickly and explaining as always! 🎉
